extensions-core/azure-extensions/src/test/java/org/apache/druid/data/input/azure/AzureEntityTest.java

```diff
@@ -28,14 +28,16 @@
 import org.apache.druid.storage.azure.AzureUtils;
 import org.easymock.EasyMock;
 import org.easymock.EasyMockSupport;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertSame;
```

Comment:
I would advise against the static imports as Druid code doesn't use this style.

Reply:
Would it be worth starting to use it more, at least in tests? I tried to look through the codebase a bit: it seems like there are sporadic usages of it, but as you said, there isn't a consistent style (say specifically for assertions). I feel the assertions read a bit more concisely this way. But I'm also happy to use a non-static import and prefix the assertion functions with the `Assertions` package name.

Reply:
The argument against static imports in general would be that they sometimes make the (non-test) code less readable. For example, a statement like `SomeObject.create()` would be written simply as `create()` which does not make for very readable/self-explanatory code.

But, I agree, that for assertions in test, we can definitely make an exception. Especially since assertion methods are appropriately named and clarify exactly what is being done in the methods, e.g. `assertTrue()`, `assertNull()`, etc. The preceding `Assert` or `Assertions` is fairly redundant.